aws-cdk-eks-app-alarms-to-slack/package.json

```diff
@@ -13,15 +13,15 @@
   "devDependencies": {
     "@types/jest": "^29.2.5",
     "@types/node": "18.11.18",
-    "aws-cdk": "2.114.1",
+    "aws-cdk": "2.132.0",
     "jest": "^29.3.1",
     "ts-jest": "^29.0.3",
     "ts-node": "^10.9.1",
     "typescript": "~4.9.4"
   },
   "dependencies": {
     "@aws-quickstart/eks-blueprints": "^1.5.4",
```

Comment:
Lets also update the blueprints version.

Reply:
updated to new BP version which was validated